components/dialog/dialog-fullscreen.js

```diff
@@ -95,12 +161,26 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 		this.async = false;
 		this._autoSize = false;
 		this._hasFooterContent = false;
+		this._iconType = 'tier1:close-large-thick';
+		this._handleResize = this._handleResize.bind(this);
+		this._mediaQueryList = window.matchMedia('(max-width: 616px)');
+		this._handleResize();
```

Comment:
Nit: this line is going to set `this._iconType` anyway, so I don't think you need to set it 2 lines up.

Reply:
The reason I added the setting iconType line was because the linter was complaining that the html could be undefined.. I can set it to an empty string?

Reply:
Oh interesting... hmm yeah or just leaving it how you have it is good! 👍